### PR TITLE
add helm chart sample

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: examples/helmchart_echo_sample/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Inference-test Sample
+This sample is a helm chart deploying `fastapi_echo_sample` container from NGC.
+
+## Prerequisites
+
+### Setup kubenetes cluster
```
Comment (sbaum1994) [inline examples/helmchart_echo_sample/README.md:6]: kubenetes 💯

File: examples/helmchart_echo_sample/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+To deploy helm chart, one needs to setup a kubenetes cluster locally. One recommendation is to use [microk8s](https://microk8s.io/docs/getting-started).
```
Comment (sbaum1994) [inline examples/helmchart_echo_sample/README.md:8]: kubenetes

File: examples/helmchart_echo_sample/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Deploying
+
+The sample helm chart will deploy a `fastapi_echo_sample` container and an `entrypoint` service for client communicating with the container.
+
+First create a secret with your staging NGC API key
+
+```
+microk8s kubectl create secret docker-registry <image-pull-secret-name> --docker-server=stg.nvcr.io --docker-username=$oauthtoken --docker-password=<STG-NGC-API-KEY>
```
Comment (sbaum1994) [inline examples/helmchart_echo_sample/README.md:22]: staging?

File: examples/helmchart_echo_sample/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+### Build and upload container image
+
+For container build and upload details, please refer to [fastapi_echo_sample](../fastapi_echo_sample/README.md).
+
+
+## Deploying
+
+The sample helm chart will deploy a `fastapi_echo_sample` container and an `entrypoint` service for client communicating with the container.
+
+First create a secret with your staging NGC API key
```
Comment (sbaum1994) [inline examples/helmchart_echo_sample/README.md:19]: NGC Personal API Key

File: examples/helmchart_echo_sample/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+```
+microk8s kubectl create secret docker-registry <image-pull-secret-name> --docker-server=stg.nvcr.io --docker-username=$oauthtoken --docker-password=<STG-NGC-API-KEY> 
+```
+
+Next deploy the sample helm chart with
+```
+microk8s helm install <release-name> /path/to/inference-test --set ngcImagePullSecretName=<image-pull-secret-name>
```
Comment (sbaum1994) [inline examples/helmchart_echo_sample/README.md:27]: dumb q - how do they get release-name or is that just whatever

**Review by sbaum1994 — APPROVED** [review]

File: examples/helmchart_echo_sample/README.md (hunk 5 above)
Comment (NVJCameron) [reply examples/helmchart_echo_sample/README.md:27]: user defined

**Pull request merged**